tutorials/linux-advanced.rst

```diff
@@ -0,0 +1,344 @@
+
+If a user falls into more than one of these scopes, their effective permissions are determined based on the first scope the user falls within in the order of user, group, and others.
+
+Each scope has three specific permissions for each file or directory:
+
+**read** - The read permission allows a user to view a file's contents. When set for a directory, this permission allows a user to view the names of files in the directory, but no further information about the files in the directory. ``r`` is shorthand for read permissions.
+
+**write** - The write permission allows a user to modify the contents of a file. When set for a directory, this permission allows a user to create, delete, or rename files. ``w`` is shorthand for write permissions.
+
+**execute** - The execute permission allows a user to execute a file (or program) using the operating system. When set for a directory, this permission allows a user to access file contents and other information about files within the directory (given that the user has the proper permissions to access the file). The execute permission does not allow the user to list the files inside the directory unless the read permission is also set. ``x`` is shorthand for execute permissions.
+
+To list information about a file, including its permissions, type::
+
+    ls -l <filepath>
+
+You'll get output of the form::
+
+    <permissions> 1 owner group <size in bytes> <date modified> <filepath>
+
+For example, if we want information on ``/usr/bin/python3.8``::
+
+    $ ls -l /usr/bin/python3.8
+    -rwxr-xr-x 1 root root 5486384 Jan 27  2021 /usr/bin/python3.8
+
+
+First thing we can notice is that the owner of the file is a user
+named ``root``. (FYI, ``root`` is a name for an account that has access
+to *all* commands and files on a Linux system.  Other accounts may
+also have "root" privileges.) The file's group is also ``root``.
+
+The permissions are ``-rwxr-xr-x``.  The initial dash (``-``)
+indicates that ``/usr/bin/python3.8`` is a file, not a directory.
+Directories have a ``d`` instead of a dash.  Then the permissions are
+listed in user, group, and others order. In this example, the owner,
+``root``, can read (``r``), write (``w``), and execute (``x``) the
+file. Users in the ``root`` group and all other users can read and
+execute the files.
+
+
+By default, any files or directories that you create will have your
+username as both the user and the group.  (If you run ``groups``,
+you'll notice that there is a group with the same name as your
+username.  You are the only member of this group.)  On our Linux
+machines, by default, new files are give read and write
+permissions to user and group and no permissions to other.  New
+directories will be set to have read, write and execute permissions
+for user and group.
+
+Exercise
+~~~~~~~~
+
+#. Verify this claim by running ``ls -l backups/copy2.txt`` and ``ls -ld  backups`` in your ``linux-tutorial-files`` directory.
```

Comment:
I've added a note in the TODO block at the top to remind myself to add some text saying that they need to download the tutorial files (but that they don't need to work through the Linux Basics tutorial to work through this one). I've also added a note in the exercise, providing the exact commands to create the necessary directory and file, in case someone didn't work through the Linux Basics tutorial.